include/btllib/nthash.hpp

```diff
@@ -1378,15 +1454,17 @@ class SeedNtHash
   unsigned get_hash_num_per_seed() const { return hash_num_per_seed; }
   unsigned get_k() const { return nthash.get_k(); }
 
-  uint64_t get_forward_hash() const { return nthash.get_forward_hash(); }
-  uint64_t get_reverse_hash() const { return nthash.get_reverse_hash(); }
+  uint64_t* get_forward_hash() const { return forward_hash; }
+  uint64_t* get_reverse_hash() const { return reverse_hash; }
 
 private:
   bool init();
 
   NtHash nthash;
   const unsigned hash_num_per_seed;
-  std::vector<SpacedSeed> seeds;
+  std::vector<SpacedSeed> blocks;
+  uint64_t* forward_hash = nullptr;
```

Comment:
Is this an array of hash values for forward and reverse? If so, should it also be present in the NtHash class? (As well as SeedNtHash).

Reply:
`forward_hash[i]` is the current forward hash for the `i`th seed (refer to `forward_hash(new uint64_t[seeds.size()])` in `SeedNtHash` constructors). In the NtHash class, this just has to be a single `uint64_t`, but here we need an array as we can have multiple seeds given to `SeedNtHash`. So it just needs to be present in `SeedNtHash`.